test_package/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
 # Instruct CMake to run moc automatically when needed
 set(CMAKE_AUTOMOC ON)
 # Create code from a list of Qt designer ui files
-set(CMAKE_AUTOUIC ON)
+set(CMAKE_AUTOUIC OFF)
```

Comment:
We don't need/build UIC.  Leaving this on and not configured correctly causes a cmake error in cmake version 3.15.x